servicedirectory/Gemfile

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source "https://rubygems.org"
+
+gem 'google-cloud-service_directory', '~> 0.2.0'
```

Comment:
could you remove the '~ 0.2.0'? As long as instructions say to use `bundle install` and not `bundle update`. That should make it so users can still use if there are breaking changes, but ci will fail

Reply:
Done.